### ISSUE TITLE
[Bug]: 'expo run:ios' fails after installing onesignal-expo-plugin

### ISSUE THREAD

**Issue by LukeV111** [OneSignal/onesignal-expo-plugin#79]
### What happened?

I have the latest version of expo-cli.

Using:

 "onesignal-expo-plugin": "^1.0.0",
 "react-native-onesignal": "^4.3.7",

### Steps to reproduce?

```Markdown
Follow the documentation found here: https://github.com/OneSignal/onesignal-expo-plugin
```

### What did you expect to happen?

Everything runs without any issues until I get to `expo run:ios`

Then I get the below error:

```
The following build commands failed:
        CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler (in target 'ExpoModulesCore' from project 'Pods')
```

### OneSignal Expo SDK version

1.0.0

### Relevant log output

```Shell
2022-04-08 15:38:39.692 xcodebuild[73298:4604834] warning:  The file reference for "OneSignalNotificationServiceExtension" is a member of multiple groups ("Pods" and ""); this indicates a malformed project.  Only the membership in one of the groups will be preserved (but membership in targets will be unaffected).  If you want a reference to the same file in more than one group, please add another reference to the same path.
** BUILD FAILED **


The following build commands failed:
        CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler (in target 'ExpoModulesCore' from project 'Pods')
        PhaseScriptExecution [CP-User]\ Generate\ Specs Library/Developer/Xcode/DerivedData/App-fspnqmnktjandhhdnfysxlewamvd/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/FBReactNativeSpec.build/Script-BD7ACB0873CA1DBF901FF4818FC9757C.sh (in target 'FBReactNativeSpec' from project 'Pods')
(2 failures)
Build logs written to app/.expo/xcodebuild.log
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jkasten2** [comment]
@LukeV111 Thanks for reporting. Could you provide the exact version of the expo and react dependencies? Share your package.json if you can. Please provide step by step or an example project if it require more than just starting a new project and adding these dependencies.

**Comment by LukeV111** [comment]
Thanks for the reply @jkasten2 

This happens if I build a completely new expo project and install the dependencies. No extra steps required.

Here is my package.json

`{
  "name": "family_budget",
  "version": "1.0.0",
  "main": "node_modules/expo/AppEntry.js",
  "scripts": {
    "start": "expo start",
    "android": "expo start --android",
    "ios": "expo start --ios",
    "web": "expo start --web",
    "eject": "expo eject"
  },
  "dependencies": {
    "expo": "~44.0.0",
    "expo-status-bar": "~1.2.0",
    "onesignal-expo-plugin": "^1.0.0",
    "react": "17.0.1",
    "react-dom": "17.0.1",
    "react-native": "0.64.3",
    "react-native-onesignal": "^4.3.7",
    "react-native-web": "0.17.1"
  },
  "devDependencies": {
    "@babel/core": "^7.12.9"
  },
  "private": true
}
`

**Comment by sfurlaneudaimon** [comment]
Hi @jkasten2, 
I confirm the bug reported by LukeV111, I am developing on a MacBook Pro (13-inch, M1, 2020) with Apple M1 chip

1. Xcode version 13.0 and 13.3.1
2. Cocoapods and gems installed: Gems/2.6.0/gems/cocoapods-1.11.3


Hope these steps and logs can help:

1. expo init testOneSignalPlugin
3. cd testOneSignalPlugin 
4. expo install onesignal-expo-plugin
5. Add onesginal config in app.json
{
  "expo": {
    "name": "testOneSignalPlugin",
    "slug": "testOneSignalPlugin",
    "version": "1.0.0",
    "orientation": "portrait",
    "icon": "./assets/icon.png",
    "splash": {
      "image": "./assets/splash.png",
      "resizeMode": "contain",
      "backgroundColor": "#ffffff"
    },
    "updates": {
      "fallbackToCacheTimeout": 0
    },
    "assetBundlePatterns": [
      "**/*"
    ],
    "ios": {
      "supportsTablet": true
    },
    "android": {
      "adaptiveIcon": {
        "foregroundImage": "./assets/adaptive-icon.png",
        "backgroundColor": "#FFFFFF"
      }
    },
    "web": {
      "favicon": "./assets/favicon.png"
    },
    "plugins": [
      [
        "onesignal-expo-plugin",
        {
          "mode": "development"
        }
      ]
    ],
    "extra": {
      "oneSignalAppId": “my-onesginal-id”
    }
  }
}



### The package.json is like this

{
  "name": "testonesignalplugin",
  "version": "1.0.0",
  "scripts": {
    "start": "expo start --dev-client",
    "android": "expo run:android",
    "ios": "expo run:ios",
    "web": "expo start --web",
    "eject": "expo eject"
  },
  "dependencies": {
    "expo": "~44.0.0",
    "expo-status-bar": "~1.2.0",
    "onesignal-expo-plugin": "^1.0.1",
    "react": "17.0.1",
    "react-dom": "17.0.1",
    "react-native": "0.64.3",
    "react-native-web": "0.17.1",
    "expo-splash-screen": "~0.14.1"
  },
  "devDependencies": {
    "@babel/core": "^7.12.9"
  },
  "private": true
}


### Then run expo run:ios with error:


Something went wrong running `pod install` in the `ios` directory.
Command `pod install` failed.
└─ Cause: Oh no, an error occurred.


Failed to build iOS project. "xcodebuild" exited with error code 65.
To view more error logs, try building the app with Xcode directly, by opening……/testOneSignalPlugin/iOS/testOneSignalPlugin.xcodeproj.

Command line invocation:
    .......Xcode13.0.app/Contents/Developer/usr/bin/xcodebuild -project ......./testOneSignalPlugin/ios/testOneSignalPlugin.xcodeproj -configuration Debug -scheme testOneSignalPlugin -destination id=BEA6BA14-1FCE-4710-B876-50088675E9AA

User defaults from command line:
    IDEPackageSupportUseBuiltinSCM = YES

note: Using new build system
note: Planning
Analyze workspace

Create build description
Build description signature: ec5b2a2fcb3540beba506ba2ea8f462e
Build description path: …./Library/Developer/Xcode/DerivedData/testOneSignalPlugin-cgxnkonryjakfedmscpzlknacshn/Build/Intermediates.noindex/XCBuildData/ec5b2a2fcb3540beba506ba2ea8f462e-desc.xcbuild

note: Build preparation complete
note: Building targets in parallel
......./testOneSignalPlugin/ios/Pods/Target Support Files/Pods-testOneSignalPlugin/Pods-testOneSignalPlugin.debug.xcconfig:1:1: error: unable to open file (in target "testOneSignalPlugin" in project "testOneSignalPlugin")
......./testOneSignalPlugin/ios/Pods/Target Support Files/Pods-testOneSignalPlugin/Pods-testOneSignalPlugin.debug.xcconfig:1:1: error: unable to open file (in target "testOneSignalPlugin" in project "testOneSignalPlugin")
......./testOneSignalPlugin/ios/Pods/Target Support Files/Pods-testOneSignalPlugin/Pods-testOneSignalPlugin.debug.xcconfig:1:1: error: unable to open file (in target "testOneSignalPlugin" in project "testOneSignalPlugin")
......./testOneSignalPlugin/ios/Pods/Target Support Files/Pods-testOneSignalPlugin/Pods-testOneSignalPlugin.debug.xcconfig:1:1: error: unable to open file (in target "testOneSignalPlugin" in project "testOneSignalPlugin")










### The log of xcode

xcodebuild[13861:222702] warning:  The file reference for "OneSignalNotificationServiceExtension" is a member of multiple groups ("Pods" and ""); this indicates a malformed project.  Only the membership in one of the groups will be preserved (but membership in targets will be unaffected).  If you want a reference to the same file in more than one group, please add another reference to the same path.
** BUILD FAILED **


Command line invocation:
    .......Xcode13.0.app/Contents/Developer/usr/bin/xcodebuild -project .......testOneSignalPlugin/ios/testOneSignalPlugin.xcodeproj -configuration Debug -scheme testOneSignalPlugin -destination id=BEA6BA14-1FCE-4710-B876-50088675E9AA

User defaults from command line:
    IDEPackageSupportUseBuiltinSCM = YES

note: Using new build system
note: Planning
Analyze workspace

Create build description
Build description signature: ec5b2a2fcb3540beba506ba2ea8f462e
Build description path: /Users/eudaimon/Library/Developer/Xcode/DerivedData/testOneSignalPlugin-cgxnkonryjakfedmscpzlknacshn/Build/Intermediates.noindex/XCBuildData/ec5b2a2fcb3540beba506ba2ea8f462e-desc.xcbuild

note: Build preparation complete
note: Building targets in parallel
.......testOneSignalPlugin/ios/Pods/Target Support Files/Pods-testOneSignalPlugin/Pods-testOneSignalPlugin.debug.xcconfig:1:1: error: unable to open file (in target "testOneSignalPlugin" in project "testOneSignalPlugin")
.......testOneSignalPlugin/ios/Pods/Target Support Files/Pods-testOneSignalPlugin/Pods-testOneSignalPlugin.debug.xcconfig:1:1: error: unable to open file (in target "testOneSignalPlugin" in project "testOneSignalPlugin")
.......testOneSignalPlugin/ios/Pods/Target Support Files/Pods-testOneSignalPlugin/Pods-testOneSignalPlugin.debug.xcconfig:1:1: error: unable to open file (in target "testOneSignalPlugin" in project "testOneSignalPlugin")
.......testOneSignalPlugin/ios/Pods/Target Support Files/Pods-testOneSignalPlugin/Pods-testOneSignalPlugin.debug.xcconfig:1:1: error: unable to open file (in target "testOneSignalPlugin" in project "testOneSignalPlugin")


I changed Xcode command line tools version to latest 13.3.1 but got the same error without logs
Something went wrong running `pod install` in the `ios` directory.
Command `pod install` failed.
└─ Cause: Oh no, an error occurred.




Thank you for your help!

**Comment by rgomezp** [comment]
Howdy,
Thanks for reporting. We'll take a look at what might be causing this issue and appreciate your patience. It also might be unrelated since there's just a warning.

I've found the following links that seem similar.
https://stackoverflow.com/questions/33395675/cocoapods-file-reference-is-a-member-of-multiple-groups
https://github.com/yonaskolb/XcodeGen/issues/782

**Comment by rgomezp** [comment]
Duplicate of #71
